src/App.tsx

```diff
@@ -1,26 +1,89 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { USER_ID, getCount, getNewTodoId, getTodos } from './api/todos';
+import { Todo, Status } from './types/Todo';
+import { TodoList } from './components/todoList';
+import { Footer } from './components/footer';
+import { ErrorNotifications } from './components/errorNotifications';
+import { Header } from './components/header';
 
 export const App: React.FC = () => {
+  const [preparedTodos, setPreparedTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [selectedFilter, setSelectedFilter] = useState<string>(Status.all);
+  const [title, setTitle] = useState<string>('');
+  const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState<number | null>(null);
```

Comment:
You need isLoading state here only true | false to use it with initial todo fetching. And also you need an own loading state inside the todoItem Component

Reply:
I don't have a loader in my task for initial todo fetching.